README.md

```diff
@@ -30,112 +30,100 @@ To facilitate robust API governance, we're excited to introduce **API Analysis**
+### :memo: API Analysis in Azure API Center (Overview)
 
 Here is a high-level overview of how API analysis works in Azure API Center
 ![Overview](./images/overview.png)
 
-## :rocket: How to set up API Analysis
+## :rocket: How to Run API Analysis
 
 ### :wrench: Configure your environment
 
 Before you get started, make sure you have the following requirements in place:
 
-- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free).
-- An API Center instance.
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio).
+- [Node.js 18.x](https://nodejs.org/en/download/releases/) or above. Use the `node --version` command to check your version.
+- [TypeScript 4.x](https://www.typescriptlang.org/). Use the `tsc -v` command to check your version.
 - [Visual Studio Code](https://code.visualstudio.com/) on one of the [supported platforms](https://code.visualstudio.com/docs/supporting/requirements#_platforms).
 - The [Azure Functions extension v1.10.4](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) or above for Visual Studio Code.
-- [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
-- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli)
+- [Azure Functions Core Tools v4.0.5382 or above](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Cisolated-process%2Cnode-v4%2Cpython-v2%2Chttp-trigger%2Ccontainer-apps&pivots=programming-language-typescript#install-the-azure-functions-core-tools).
+- [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd).
+- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli).
 
-### :cloud: How to deploy
-
-Follow the instructions in [Quickstart: Create a function in Azure with TypeScript using Visual Studio Code](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-typescript?pivots=nodejs-model-v4#sign-in-to-azure). Start from the "Sign in to Azure" section and complete all subsequent sections.
+### :wrench: Running the sample using the Azure Developer CLI (azd)
 
-Alternatively, if you'd like to use [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd), run the following steps:
+The Azure Developer CLI (`azd`) is a developer-centric command-line interface (CLI) tool for creating Azure applications.
 
-1. Login to Azure by running the commands below in your terminal:
+After logging in with the following command, you will be able to use the `azd` cli to quickly provision and deploy the application.
 
-    ```bash
-    az login
-    azd auth login
-    ```
+```
+# authenticate to Azure Developer CLI
+azd auth login
```

Comment:
Please add the predown and preprovision hooks that login the user following the pattern found here: 

https://github.com/jongio/hello-ai/blob/main/infra/hooks/predown.sh
https://github.com/jongio/hello-ai/blob/main/infra/hooks/preprovision.sh
https://github.com/jongio/hello-ai/blob/main/infra/hooks/login.sh

You'll need to create for both ps1 and sh for this project.  Once you have that working, please do a PR to hello-ai with any changes, including the ps1 versions.